src/view/login.js

```diff
@@ -0,0 +1,51 @@
+import Layout from "component/layout";
+
+var getUser = function(acct_id, users) {
+    var acctUser = users.filter(function(u) { return u.accountId() == acct_id; });
```

Comment:
What standard do we want to use for variables/function names? Have we looked into what is standard for JS. Ha! Standards in Javascript... :laughing: